Doc/whatsnew/3.10.rst

```diff
@@ -234,6 +234,12 @@ Deprecated
 Removed
 =======
 
+* Removed special methods ``__int__``, ``__float__``, ``__floordiv__``,
+  ``__mod__``, ``__divmod__``, ``__rfloordiv__``, ``__rmod__`` and
+  ``__rdivmod__`` of the :class:`complex` class.  They always raised
+  a :exc:`TypeError`.
```

Comment:
It's unclear if it changes the behavior. Maybe explain that these operations still exist and still raise TypeError?

Reply:
Sorry, I missed this comment. Could you propose new wording? Or maybe you want to make the documentation PR yourself?

Reply:
It doesn't matter. Leave it as it is :-)